### ISSUE TITLE
Question: Still possible to generate Inline Response Schemas with Widdershins 3.6.0?

### ISSUE THREAD

**Issue by kkwanpsn** [Mermade/widdershins#172]
Previously in ver. 3.4, when working with external references, using the `--resolve` option generated expanded request/response schemas inline. Some markdown for example:

```
<h3 id="listpets-responses">Responses</h3>

|Status|Meaning|Description|Schema|
|---|---|---|---|
|200|[OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)|A paged array of pets|Inline|
|default|Default|unexpected error|Inline|

<h3 id="listpets-responseschema">Response Schema</h3>

Status Code **200**

|Name|Type|Required|Restrictions|Description|
|---|---|---|---|---|
|» id|integer(int64)|true|none|none|
|» name|string|true|none|none|
|» tag|string|false|none|none|

Status Code **default**

|Name|Type|Required|Restrictions|Description|
|---|---|---|---|---|
|» code|integer(int32)|true|none|none|
|» message|string|true|none|none|

```

Here is that same markdown rendered using Slate for additional reference:

![screen shot 2018-09-13 at 3 56 48 pm](https://user-images.githubusercontent.com/26638305/45577481-fe593400-b830-11e8-9347-949587ce2520.png)

Is this still possible using Widderhins 3.6?

**Comment by MikeRalphson** [comment]
Yes, as far as I recall no features have been deliberated removed. Please chase if you're getting different results with the latest version.

**Comment by MikeRalphson** [comment]
Bug, fixed in`v3.6.3` - see #175
